kratos/tests/cpp_tests/geometries/test_line_3d_2.cpp

```diff
@@ -314,6 +314,18 @@ namespace Testing {
         CrossCheckShapeFunctionsValues(*p_geom_nodes);
     }
 
+    KRATOS_TEST_CASE_IN_SUITE(Line3D2ShapeFunctionsValues, KratosCoreGeometriesFastSuite) {
+        auto geom = GeneratePointsDiagonalLine3D2();
+        array_1d<double, 3> coord(3);
+        coord[0] = 2.0/3.0;
+        coord[1] = 2.0/3.0;
+        coord[2] = 2.0/3.0;
+        Vector N_values;
+        geom->ShapeFunctionsValues(N_values, coord);
```

Comment:
What I was trying to do was to call `const Matrix& n_container = geom->ShapeFunctionsValues( GeometryData::GI_GAUSS_2 );` at this location (line 324). Shouldn't this be possible?